fog/overseer/server/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:20.04
```

Comment:
I think we normally specify a specific sha for this. Looking at the other dockerfiles we have in fullservice or mobilecoin.